pkg/project/registry/rest.go

```diff
@@ -0,0 +1,68 @@
+package registry
```

Comment:
Yes, strategy is common code that applies to API objects independent of their persistence mechanisms.

----- Original Message -----
> > @@ -0,0 +1,68 @@
> > +package registry
> 
> Aligning with upstream pattern:
> 
> https://github.com/GoogleCloudPlatform/kubernetes/tree/master/pkg/registry/namespace
> 
> Note the strategy as I understand is intended to be shared across different
> persistence options (etcd / proxy / etc.)
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/openshift/origin/pull/1601/files#r27891807